test/IRGen/section.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %target-swift-frontend -primary-file %s -emit-sil | %FileCheck %s --check-prefix=SIL
+// RUN: %target-swift-frontend -primary-file %s -emit-ir  | %FileCheck %s --check-prefix=IR
```

Comment:
Might be one of the few places that it actually makes sense to check the assembly too :)

Reply:
Just to make sure that it makes it all the way through